satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -1404,9 +1525,11 @@ datasets:
     wavelength: [0.384, 0.444, 0.504]
     file_type: nc_fci_asr
     file_key: reflectance_mean
```

Comment:
Please also add `units: %` since it's set to "none" in the netcdf file.

**This applied to all reflectance entries using the `file_key: reflectance_mean`**

Reply:
This is actually missing for all reflectance & quality variables